src/main/resources/templates/admin/defaultVoters.html

```diff
@@ -34,6 +41,16 @@ <h2>Default Voters</h2>
                 <input type="submit" value="Remove"/>
             </form>
         </td>
+        <td> </td>
+        <td th:each="project : ${projects}">
+            <input type="checkbox"
+                   name="projectIds"
+                   th:form="|remove-${user.userId}|"
+                   th:value="|${project.projectId}|"
+                   th:title="|${project.projectId}|"
+                   th:checked="${project.voteDecision == null}"
```

Comment:
Am I correct that the checkbox for a given user on a given project's row doesn't reflect whether or not the user is currently a voter on that project? I feel like that's maybe a bit counterintuitive since the grid layout suggests to me that I'm seeing per-user, per-project values. Maybe others will disagree with me on this, though.

Reply:
I am thinking of folding this into a manage voters screen. And when a user gets assigned to be default voters, instead of propagating to some subsets of projects, we will show that in the table UI directly. Better for clarity I believe. I am putting out a PR right now for that.